gnoi/os/server.go

```diff
@@ -25,15 +22,19 @@ import (
 // Server is an OS Management service.
 type Server struct {
 	pb.OSServer
-	manager *Manager
+	manager               *Manager
+	activationFailMessage string
```

Comment:
probably better to have this in manager, it avoids spliting the logic between the files.
server is more like a grpc abstraction
manager is more like the os abstraction

Reply:
That makes sense! Will do